assets/javascripts/discourse/widgets/question-answer-user-commented-notification-item.js

```diff
@@ -0,0 +1,21 @@
+import { DefaultNotificationItem } from "discourse/widgets/default-notification-item";
+import { createWidgetFrom } from "discourse/widgets/widget";
+import { iconNode } from "discourse-common/lib/icon-library";
+import { postUrl } from "discourse/lib/utilities";
+import { buildAnchorId } from "../widgets/qa-comment";
+
+createWidgetFrom(
+  DefaultNotificationItem,
+  "question-answer-user-commented-notification-item",
+  {
+    url(data) {
+      const attrs = this.attrs;
+      const url = postUrl(attrs.slug, attrs.topic_id, attrs.post_number);
+      return `${url}#${buildAnchorId(data.qa_comment_id)}`;
```

Comment:
Note that we only load 5 comments initially per post. If the notification is for the sixth's comment, we will just highlight the current post for now. I intend to improve on this after the current PR.